pkg/operator/ceph/controller/predicate.go

```diff
@@ -747,15 +750,23 @@ func isDeployment(obj runtime.Object, appName string) bool {
 	return false
 }
 
-func isCMTConfigOverride(obj runtime.Object) bool {
+func isCMTConfigOverride(obj runtime.Object) (bool, bool) {
```

Comment:
how about passing both the new and existing object to this function and compare the `data` part of the config map?

If data is same, then don't reconcile, else reconcile.

Reply:
Yes I'll make that change to help catch the case of it being changed to empty. Not sure it's worth a deep comparison of the map, but will take a look.

Reply:
Actually, using `reflect.DeepEqual()` keeps it very simple